etc/license.txt

```diff
@@ -1,13 +1,5 @@
- Copyright 2017 - 2023 The ModiTect authors
+ SPDX-License-Identifier: Apache-2.0
 
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
+  Copyright The original authors
```

Comment:
This line and the next have one extra space of indentation. Please make it only one space in total and reformat all files once again. Thank you. 😅

Reply:
It was on purpose of course: to check the hawk eye of the reviewer 😉. You passed the test!
Thanks Andres!